userspace/libsinsp/sinsp.cpp

```diff
@@ -2851,9 +2857,6 @@ void sinsp::handle_plugin_async_event(const sinsp_plugin& p, std::unique_ptr<sin
 		auto plid = (uint32_t*)((uint8_t*) evt->m_pevt + sizeof(scap_evt) + 4+4+4);
 		*plid = cur_plugin_id;
 		evt->inspector(this);
-		evt->m_pevt->ts = (m_lastevent_ts == 0)
```

Comment:
For now, this should forcefully be:
```cpp
evt->m_pevt->ts = (uint64_t) -1;
```

Controlling the timestamp from the plugin would be a feature or a bug right now. We may consider that for the future.